src/Symfony/Component/Console/Tester/ApplicationTester.php

```diff
@@ -97,27 +101,15 @@ public function run(array $input, $options = array())
             $streamProperty->setValue($this->output, fopen('php://memory', 'w', false));
         }
 
-        return $this->statusCode = $this->application->run($this->input, $this->output);
-    }
+        $this->statusCode = $this->application->run($this->input, $this->output);
 
-    /**
-     * Gets the display returned by the last execution of the application.
-     *
-     * @param bool $normalize Whether to normalize end of lines to \n or not
-     *
-     * @return string The display
-     */
-    public function getDisplay($normalize = false)
-    {
-        rewind($this->output->getStream());
-
-        $display = stream_get_contents($this->output->getStream());
+        putenv('SHELL_INTERACTIVE');
```

Comment:
`putenv($shellInteractive ? "SHELL_INTERACTIVE=$shellInteractive" : 'SHELL_INTERACTIVE');` instead, removing the condition below